server/init_topic.go

```diff
@@ -113,7 +113,19 @@ func topicInit(t *Topic, sreg *sessionJoin, h *Hub) {
 func initTopicMe(t *Topic, sreg *sessionJoin) error {
 	t.cat = types.TopicCatMe
 
-	user, err := store.Users.Get(sreg.sess.uid)
+	uid := types.ParseUserId(t.name)
+	if uid.IsZero() {
+		// Log out the session
+		sreg.sess.uid = types.ZeroUid
+		log.Println("malformed topic name (sub me): ", t.name, sreg.sess.sid)
+		return types.ErrMalformed
+	}
+	if uid != sreg.sess.uid && sreg.sess.authLvl != auth.LevelRoot {
```

Comment:
I don't understand this change. Maybe it's not needed.

Issuing a command `onBehalfOf` is only available to root user:
https://github.com/tinode/chat/blob/master/server/session.go#L277
`t.name` is assigned here:
https://github.com/tinode/chat/blob/master/server/session.go#L1030
and it matches `onBahalfOf`.

Reply:
Ah, good. Thanks, removed.